src/providers/Onboarding.js

```diff
@@ -0,0 +1,113 @@
+import React, { useCallback, useContext, useState } from 'react'
+import PropTypes from 'prop-types'
+import { Screens } from '@components/Onboarding/Screens/config'
+
+const OnboardingContext = React.createContext()
+
+const stepsLength = Screens.length
+
+const DEFAULT_CONFIG = {
```

Comment:
Default values, we still need to define what defaults we'll use for some of them

Reply:
Also, we should agree on the naming for each to align with the latest discussions.